src/main/java/uk/gov/indexer/fetchers/EntriesFunction.java

```diff
@@ -0,0 +1,9 @@
+package uk.gov.indexer.fetchers;
+
+import uk.gov.indexer.dao.Entry;
+
+import java.util.List;
+
+public interface EntriesFunction{
```

Comment:
Maybe EntryGetter is a better name? Would lead to "return object.entryGetter().get()" ?